Community/kavSimulations/config/KavSim Overhead Board Config v1-0-0.mfmc

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ModuleName>Kav Mega Ovrhd</ModuleName>
```

Comment:
Same here. Drop the Mega... no need to have that in the name.
Add ModuleType.

Reply:
ModuleName changed and ModuleType added